### PR TITLE
Buttons

### PR DESCRIPTION
**Pull request by vaibhav-mahalle**

### CODE REVIEW

File: components/button.css [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    border-radius: 50%;
+    border: none;
+    background-color: var(--primary-100);
+}
+
+
+
+:root{
+    --primary-100: rgb(23, 196, 185);
+    --primary-100: hsl(177, 100%, 16%);
```
Comment (vivekmehto) [inline components/button.css]: same variable declartions. avoid these.

File: components/button.css [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    height: 3rem;
+    width: 3rem;
+    border-radius: 50%;
+    border: none;
+    background-color: var(--primary-100);
+}
+
+
+
+:root{
```
Comment (vivekmehto) [inline components/button.css]: variable declarations in root should be on the bottom. Its good practice to declare these on top

File: components/button.css [hunk 3]
```diff
@@ -0,0 +1,128 @@
+@import url(/stylesheets/style.css);
+
+
+.page-container{
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+}
+
+.container{
```
Comment (vivekmehto) [inline components/button.css]: The container class should be in utilities CSS file

File: components/button.css [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+.outlined-link{
+    text-decoration: none;
+    color: var(--link-color);
+    background-color: white;
+    padding: 0.6rem 1rem;
+    border: 1px solid ;
+}
+
+.icon{
```
Comment (vivekmehto) [inline components/button.css]: You can use btn-icon to be more specific.

File: components/button.css [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+}
+
+.btn-link{
+    text-decoration: none;
+    color: white;
+    background-color: var(--link-color);
+    padding: 0.6rem 1rem;
+}
+
+.outlined-primary{
```
Comment (vivekmehto) [inline components/button.css]: You can use btn-outline-primary in this also it would be more readable to users.

File: components/button.css [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+    padding: 0.8rem 2rem;
+    border-radius: 6px;
+    border: none;
+    outline: none;
+    cursor: pointer;
+
+}
+
+.btn-primary{
+    color: white;
```
Comment (vivekmehto) [inline components/button.css]: color: white is repeating more than 5-6 times. Avoid using repeated code.

**Review by vivekmehto — COMMENTED** [review]
Please provide a preview of the website. It's hard to review.

File: Documentation/landing.css [hunk 7]
```diff
@@ -0,0 +1,12 @@
+@import url(/stylesheets/style.css);
+
+.container{
+    width: 100%;
+
```
Comment (sakleprajwal) [inline Documentation/landing.css:5]: Avoid spaces

File: Documentation/landing.html [hunk 8]
```diff
@@ -2,6 +2,9 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@300&display=swap" rel="stylesheet">
```
Comment (sakleprajwal) [inline Documentation/landing.html:7]: Give proper indentation

**Review by Sameony — APPROVED** [review]

File: components/button.css (hunk 2 above)
Comment (ttahm3d) [inline components/button.css]: move variables to common css file

File: components/button.html [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="button.css" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (ttahm3d) [inline components/button.html]: Change the name to something like Docs | UI Library name

**Review by ttahm3d — COMMENTED** [review]
Don't use same commit messages for more than one commit. Include changes as others have mentioned

File: Documentation/landing.html (hunk 8 above)
Comment (vaibhav-mahalle) [reply Documentation/landing.html:7]: resolved the issue, in latest commit.

File: components/button.css (hunk 2 above)
Comment (vaibhav-mahalle) [reply components/button.css]: resolved the issue in latest commit. Please check and suggest improvements.

File: components/button.html (hunk 9 above)
Comment (vaibhav-mahalle) [reply components/button.html]: resolved the issue in latest commit. Please check and suggest improvements.

**Review by ttahm3d — APPROVED** [review]
Looks good

**Comment by vaibhav-mahalle** [comment]
made relative changes and approved from the required reviewers.

**Pull request merged**